.gitlab-ci.yml

```diff
@@ -140,35 +141,17 @@ default:
     - if: $CI_COMMIT_REF_NAME == "master"
     - if: $CI_COMMIT_REF_NAME =~ /^v[0-9]+\.[0-9]+.*$/              # i.e. v1.0, v2.1rc1
 
-.nightly-pipeline:                 &nightly-pipeline
+.nightly-pipeline:
   rules:
     # this job runs only on nightly pipeline with the mentioned variable, against `master` branch
     - if: $CI_COMMIT_REF_NAME == "master" && $CI_PIPELINE_SOURCE == "schedule" && $PIPELINE == "nightly"
 
-.merge-ref-into-master-script:     &merge-ref-into-master-script
-  - if [ $CI_COMMIT_REF_NAME != "master" ]; then
-      git fetch origin +master:master;
-      git fetch origin +$CI_COMMIT_REF_NAME:$CI_COMMIT_REF_NAME;
-      git checkout master;
-      git config user.email "ci@gitlab.parity.io";
-      git merge $CI_COMMIT_REF_NAME --verbose --no-edit;
-    fi
-
-.cargo-check-benches-script:       &cargo-check-benches-script
-  - mkdir -p ./artifacts/benches/$CI_COMMIT_REF_NAME-$CI_COMMIT_SHORT_SHA
-  - SKIP_WASM_BUILD=1 time cargo +nightly check --benches --all
-  - 'cargo run --release -p node-bench -- ::node::import::native::sr25519::transfer_keep_alive::paritydb::small --json
-    | tee ./artifacts/benches/$CI_COMMIT_REF_NAME-$CI_COMMIT_SHORT_SHA/::node::import::native::sr25519::transfer_keep_alive::paritydb::small.json'
-  - 'cargo run --release -p node-bench -- ::trie::read::small --json
-    | tee ./artifacts/benches/$CI_COMMIT_REF_NAME-$CI_COMMIT_SHORT_SHA/::trie::read::small.json'
-  - sccache -s
-
-
 #### stage:                       .pre
 
 skip-if-draft:
-  image:                           paritytech/tools:latest
-  <<:                              *kubernetes-env
+  extends:                         .kubernetes-env
```

Comment:
`extends:` has to be used instead of `<<: *anchor-name` notation.
Just documenting here.